src/components/PatchActionButtons/RestartPatch.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from "react";
 import { Button } from "components/Button";
 import { DropdownItem } from "components/ButtonDropdown";
-import { PatchRestartModal } from "pages/patch/index";
+import { PatchRestartModal } from "pages/version/PatchRestartModal";
```

Comment:
if this restart modal is now going to apply to both patches and versions, maybe it should just be called RestartModal (same with other things that use this naming convention; I'm also not sure if that's what's happening here though). Or VersionRestartModal. Patches can't actually technically be restarted since they are the unfinalized representation of a version.